Sofa/framework/Core/src/sofa/core/visual/VisualModel.cpp

```diff
@@ -54,6 +54,32 @@ void VisualModel::drawVisual(const VisualParams* vparams)
     doDrawVisual(vparams);
 }
 
+void VisualModel::updateVisual(const VisualParams* vparams)
```

Comment:
I have a deep feeling of cleaness when I see this kind of refactoring (factoring things in the base class)